plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeTransactionLogEntry.java

```diff
@@ -135,9 +145,16 @@ public CommitInfoEntry getCommitInfo()
         return commitInfo;
     }
 
+    @Nullable
+    @JsonProperty
+    public CDFFileEntry getCDC()
```

Comment:
Because for some strange reason files entries in delta lake metadata that contains files related to CDF are named `cdc` and looks like this:
```
{"cdc":{"path":"_change_data/cdc-00000-747c7edb-7ae7-499c-b914-71680ca2f7e9.c000.snappy.parquet","partitionValues":{},"size":1431,"dataChange":false}}
```